config/rubocop.yml

```diff
@@ -36,5 +36,8 @@ Style/StringLiteralsInInterpolation:
 Style/TrailingCommaInArguments:
   Enabled: false
 
-Style/TrailingCommaInLiteral:
+Style/TrailingCommaInArrayLiteral:
+  Enabled: false
+
+Style/TrailingCommaInHashLiteral:
```

Comment:
@ypresto Thanks for your contribution! But it just seems a bit oddly.

The `Style/TrailingCommaInArrayLiteral` and `Style/TrailingCommaInHashLiteral` cops are introduced on RuboCop 0.53.0, but we use the RuboCop v0.52.1 via onkcop.

* https://github.com/bbatsov/rubocop/blob/master/CHANGELOG.md#0530-2018-03-05
* https://github.com/codetakt/rubocop-codetakt/blob/v0.8.3/rubocop-codetakt.gemspec#L15
* https://github.com/onk/onkcop/blob/v0.52.1.1/onkcop.gemspec#L22

What version of RuboCop are you using?